product_supplierinfo_for_customer/README.rst

```diff
@@ -27,7 +27,7 @@ Product Supplierinfo for Customers
 
 This modules allows to use supplier info structure, available in
 *Inventory* tab of the product form, also for defining customer information,
-allowing to define prices per customer and product.
+allowing to define "Customer Product Name" and "Customer Product Code".
```

Comment:
Well, you can define prices for the customer, even if they are not used elsewhere. If we remove this here I'll put something in the Known Issues section like "The prices and minimum quantitties defined there are just information"

Reply:
See my screenshot, you cannot :thinking: 
https://user-images.githubusercontent.com/8736623/51282091-8d937580-19e4-11e9-8955-e590f73a7982.png